rake/spec/default/50-installed-cli-versions_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe 'st2 cli version checks' do
+  describe command("st2 --version") do
+    its(:exit_status) { is_expected.to eq 0 }
+    # TODO: Show version number in Rspec output
+    puts described_class.stdout
+  end
+
+  if spec[:mistral_enabled]
+    describe command("mistral --version") do
+      its(:exit_status) { is_expected.to eq 0 }
+      # TODO: Show version number in Rspec output
```

Comment:
@dennybaa I couldn't figure out how to output `st2 --version` stdout in Rspec tests.
Leaving it as TODO.

Maybe you know. If you'll have time to add it - It would be nice :cake: